src/header/header.scss

```diff
@@ -266,3 +271,70 @@
     }
   }
 }
+
+.iui-split-menu-header-button {
```

Comment:
Why do we need a completely separate new class for this? I thought we were making all the header buttons split.

Reply:
I don't think it's a good idea to make ALL header buttons split. 
I can remove separate class and move functionality the existing one.

Reply:
I think we should support both.. Both are valid.
@FlyersPh9

Reply:
I think most workflows will use the split menu button.  I can't think of when they'd want the single button, but maybe we support it just in case?

Reply:
In either case, there can be a good bit of code reuse (by making this class a _modifier_ on the normal header button). I think will help fix the specificity problems too.

Reply:
Moved css class.